src/docs/common/snippets/common-mainclass.adoc

```diff
@@ -0,0 +1,23 @@
+:exclude-for-build:gradle
+
+[source, xml]
+.pom.xml
+----
+...
+    <properties>
+...
```

Comment:
The dots look weird in the generated docs. I'd just remove them

Reply:
I prefer them to be clear there is content more missing